src/objects/config.ts

```diff
@@ -15,23 +16,17 @@ const defaults = {
       background: '#ffffff',
     }),
   },
-} as const satisfies Omit<RequiredGraphConfig, 'runId' | 'fetch'>
+}
 
 function withDefaults(config: RunGraphConfig): RequiredGraphConfig {
-  return {
-    runId: config.runId,
-    fetch: config.fetch,
-    animationDuration: config.animationDuration ?? defaults.animationDuration,
-    nodeRenderKey: config.nodeRenderKey ?? defaults.nodeRenderKey,
-    styles: {
-      nodeHeight: config.styles?.nodeHeight ?? defaults.styles.nodeHeight,
-      nodeMargin: config.styles?.nodeMargin ?? defaults.styles.nodeMargin,
-      node: node => ({
-        ...defaults.styles.node(),
-        ...config.styles?.node?.(node),
-      }),
-    },
-  }
+  const value: RequiredGraphConfig = merge({}, defaults, config)
+
+  value.styles.node = thing => ({
+    ...defaults.styles.node(thing),
+    ...config.styles?.node?.(thing),
+  })
```

Comment:
```suggestion
  value.styles.node = node => ({
    ...defaults.styles.node(node),
    ...config.styles?.node?.(node),
  })
```
vestigial from pairing 😬

Reply:
Lol! Thank you